misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/integration/ChronoLocalDateAssertionsSpec.kt

```diff
@@ -0,0 +1,47 @@
+package ch.tutteli.atrium.specs.integration
+
+
+import ch.tutteli.atrium.api.fluent.en_GB.messageContains
+import ch.tutteli.atrium.api.fluent.en_GB.toThrow
+import ch.tutteli.atrium.api.verbs.internal.expect
+import ch.tutteli.atrium.specs.*
+import ch.tutteli.atrium.translations.DescriptionDateTimeLikeAssertion
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+import java.time.LocalDate
+import java.time.chrono.ChronoLocalDate
+
+abstract class ChronoLocalDateAssertionsSpec(
+    //isBefore, isBeforeEquals, isAfter
+    isAfterOrEquals: Fun1<ChronoLocalDate, ChronoLocalDate>,
+    describePrefix: String = "[Atrium] "
+) : Spek({
+
+    include(object : SubjectLessSpec<ChronoLocalDate>(
+        describePrefix,
+        isAfterOrEquals.forSubjectLess(1)
+    ) {})
+
+    val isAfterOrEqualsDescr = DescriptionDateTimeLikeAssertion.IS_AFTER_OR_EQUALS.getDefault()
+
+    val fluent = expect(LocalDate.of(2019, 12, 15) as ChronoLocalDate)
+
+    describe("$describePrefix context subject is 2019/12/15") {
+        describe("${isAfterOrEquals.name} ...") {
+            val isAfterOrEqualsFun = isAfterOrEquals.lambda
+
+            it("... 2019/12/16 throws an AssertionError containing ${DescriptionDateTimeLikeAssertion::class.simpleName}.${DescriptionDateTimeLikeAssertion.IS_AFTER_OR_EQUALS} and `: 11`") {
```

Comment:
IMO good enough if you write only
```suggestion
            it("... 2019/12/16 throws an AssertionError") {
```
if you want to keep it this way, then please fix `: 11`, should be `: 2019/12/16`